### PR TITLE
Feature/cldn 1118 trino upgrade

### PR DESCRIPTION
**Pull request by cccs-mmrouet**
Updated our trino branch with the latest from master and updated;
extra certs installation, documentation, image building pipeline and folder structure in container.

### CODE REVIEW

File: run-trino-env.sh [hunk 1]
```diff
@@ -1,17 +1,17 @@
-#!/bin/sh
+#!/bin/bash
 
-trinoCloneDir=${1:-~/repositories/trino}
+trinoCloneDir=${1:-~/repositories/cccs-work/trino}
```
Comment (cccs-tom) [inline run-trino-env.sh:3]: Is this cccs-work folder structure widely-spread enough that we want to make it the default for everyone? I certainly don't use it...

File: README.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
 
+## Trino specific instructions on how to run the container:
+Run a Docker container with Projector and JetBrains IDE.
+
+### run-trino-env.sh
+ This is the only script you need to run, everything else is taken care of by DevOps with the azure pipeline.
+ This script will connect you to UChimera and give you acccess to our custom image.
+ 
+ ```shell script
+./run-trino-env.sh
```
Comment (cccs-tom) [inline README.md]: We should document the optional argument to the script (the path where the Trino repo is cloned).

**Review by cccs-tom — CHANGES_REQUESTED** [review]

File: run-trino-env.sh (hunk 1 above)
Comment (cccs-mmrouet) [reply run-trino-env.sh:3]: Well that is the structure recommended in the "So You're new to Dasi2C"  document!

File: README.md (hunk 2 above)
Comment (cccs-mmrouet) [reply README.md]: Will do.

File: run-trino-env.sh (hunk 1 above)
Comment (cccs-tom) [reply run-trino-env.sh:3]: Ah! I did not realize that. I'm also not sure why... I don't do anything on my laptop that *wouldn't* qualify as "cccs-work". But ok! 🤷

File: README.md (hunk 2 above)
Comment (cccs-mmrouet) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -7,7 +7,9 @@ Run a Docker container with Projector and JetBrains IDE.
 
 ### run-trino-env.sh
  This is the only script you need to run, everything else is taken care of by DevOps with the azure pipeline.
- This script will connect you to UChimera and give you acccess to our custom image.
+ This script will connect you to UChimera and give you acccess to our custom image.  Please note that the script takes an optional
+ argument that is the location where the trino repository was cloned, should you have a folder structure that differs from the standard
+ "~/repositories/cccs-work/trino".
  
  ```shell script
 ./run-trino-env.sh
```
Comment (cccs-tom) [inline README.md]:
```suggestion
./run-trino-env.sh [path to clone]
```

**Review by cccs-tom — CHANGES_REQUESTED** [review]

**Review by cccs-tom — APPROVED** [review]

**Pull request merged**